<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>metaVariableEditor</class>
 <widget class="QWidget" name="metaVariableEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>57</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>57</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="variableName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Variable name</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="valueSource">
     <property name="toolTip">
      <string>Source of variable data</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="xminLabel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;x&lt;span style=&quot; vertical-align:sub;&quot;&gt;min&lt;/span&gt;=&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>xmin</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="xmin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Minimum &lt;i&gt;x&lt;/i&gt; value for variable data</string>
       </property>
       <property name="text">
        <string>-inf</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="xmax">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Maximum &lt;i&gt;x&lt;/i&gt; value for variable data</string>
       </property>
       <property name="text">
        <string>inf</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="xmaxLabel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;x&lt;span style=&quot; vertical-align:sub;&quot;&gt;max&lt;/span&gt;=&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="buddy">
        <cstring>xmax</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="useIndex">
     <property name="toolTip">
      <string>Use only some of the connected data items (by index)</string>
     </property>
     <property name="text">
      <string>Select
indexes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="indexBeginLabel">
       <property name="text">
        <string>begin</string>
       </property>
       <property name="buddy">
        <cstring>indexBegin</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="indexBegin">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="indexIncrementLabel">
       <property name="text">
        <string>increment</string>
       </property>
       <property name="buddy">
        <cstring>indexIncrement</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="indexIncrement">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="indexEndLabel">
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>end</string>
       </property>
       <property name="buddy">
        <cstring>indexEnd</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="indexEnd">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>9999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff/>
      </iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>variableName</tabstop>
  <tabstop>valueSource</tabstop>
  <tabstop>xmin</tabstop>
  <tabstop>useIndex</tabstop>
  <tabstop>indexBegin</tabstop>
  <tabstop>indexIncrement</tabstop>
  <tabstop>indexEnd</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>useIndex</sender>
   <signal>toggled(bool)</signal>
   <receiver>indexBeginLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useIndex</sender>
   <signal>toggled(bool)</signal>
   <receiver>indexIncrementLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useIndex</sender>
   <signal>toggled(bool)</signal>
   <receiver>indexEndLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useIndex</sender>
   <signal>toggled(bool)</signal>
   <receiver>indexEnd</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useIndex</sender>
   <signal>toggled(bool)</signal>
   <receiver>indexIncrement</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useIndex</sender>
   <signal>toggled(bool)</signal>
   <receiver>indexBegin</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>metaVariableEditor</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xmin</sender>
   <signal>editingFinished()</signal>
   <receiver>metaVariableEditor</receiver>
   <slot>sortMinMaxValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xmax</sender>
   <signal>editingFinished()</signal>
   <receiver>metaVariableEditor</receiver>
   <slot>sortMinMaxValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sortMinMaxValues()</slot>
 </slots>
</ui>
